MyLibrary/Sources/DataClient/Client.swift

```diff
@@ -10,23 +10,29 @@ public struct DataClient {
   public var fetchWorkshop: @Sendable () throws -> Conference
   public var fetchSponsors: @Sendable () throws -> Sponsors
   public var fetchOrganizers: @Sendable () throws -> [Organizer]
+  public var saveDay1: @Sendable (Conference) throws -> Void
+  public var saveDay2: @Sendable (Conference) throws -> Void
+  public var saveWorkshop: @Sendable (Conference) throws -> Void
```

Comment:
It's better to create UserDefaultsClient instead of DataClient since this will be changed to ApiClient for example.
Also, favorite data is better to store in Document folder instead of User Default. What do you think?

Reply:
@d-date I created `FileClient` in 23983b585d6da422a944416d2e44475a55d26ce8 and change save/load area to Document in 0e8b40340006e6aba6054a936211d851674df4ba